src/settings/RowComponent.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import {
+  Col,
+  Row,
+  Tooltip,
+} from '@folio/stripes/components';
+
+export default class RowComponent extends Component {
+  constructor(props) {
+    super(props);
+
+    this.rowComponent = React.createRef();
+  }
+
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    tooltip: PropTypes.string,
+    // Additional options for select box
+    dataOptions: PropTypes.array,
+    component: PropTypes.node.isRequired,
+  };
+
```

Comment:
Please add defaultProps to not required fields

Reply:
Fixed